proto/iotics/api/twin.proto

```diff
@@ -58,7 +58,7 @@ message Twin {
 // List all twins.
 message ListAllTwinsRequest {
   Headers headers = 1;
-  // Limit the results according to the value (optional: when not supplied, assume no default limits required - platform specific).
+  // Request range
```

Comment:
Search says: "Search request range"

.. would "Listing result range" be more applicable?

Reply:
https://github.com/Iotic-Labs/api/pull/10/commits/bc0f8fbc37582bd2a44de7b9caa16848389b4cda :+1: